llvm/test/CodeGen/AArch64/GlobalISel/store-merging-debug.mir

```diff
@@ -86,8 +86,8 @@ frameInfo:
   hasMustTailInVarArgFunc: false
   hasTailCall:     false
   localFrameSize:  0
-  savePoint:       ''
-  restorePoint:    ''
+  savePoints:      []
```

Comment:
You will cut out a huge amount of spurious diff if you teach the parser to support both the singular and plural forms here.